modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -33,7 +33,7 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
 						<%= contentDashboardAdminDisplayContext.getAuditGraphTitle() %>
 					</h2>
 
-					<div id="audit-graph">
+					<div id="audit-graph" class="audit-graph">
```

Comment:
I think the `id` is not been used anywhere. Maybe you can use it for the CSS instead of adding a new class with the same name. Or maybe just keep the class. I don't know, up to you.